File: solutions/3307.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int n;
    cin >> n;
    while (n--) {
        double a;
        cin >> a;
        double r = sqrt(a / (4.0 * 3.14));
        if (r < 12) {
            printf("vermelho = R$ %.2f\n", a * 0.09);
        } else if (r >= 12 && r <= 15) {
            printf("azul = R$ %.2f\n", a * 0.07);
        } else {
            printf("amarelo = R$ %.2f\n", a * 0.05);
        }
    }

    return 0;
}

File: solutions/3307.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n').slice(1, -1);

lines.forEach(num => {
  let a = parseFloat(num);
  let r = Math.sqrt(a / (4.0 * 3.14));
  if (r < 12) {
    console.log("vermelho = R$", (a * 0.09).toFixed(2));
  } else if (r >= 12 && r <= 15) {
    console.log("azul = R$", (a * 0.07).toFixed(2));
  } else {
    console.log("amarelo = R$", (a * 0.05).toFixed(2));
  }
});
